src/screens/Questions.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import { View } from 'react-native';
+import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
+import DescriptiveQ from '../components/DescriptiveQ';
+import TimesQ from '../components/TimesQ';
+
+const questions = [
```

Comment:
Maybe we can have this as an array of objects instead of array of array of objects